### ISSUE TITLE
Create Project TS error preventing both path and name from being specified

### ISSUE THREAD

**Issue by kingmesal** [jdalrymple/gitbeaker#3527]
**Description**

- Gitbeaker version: 39.34.0
- Gitbeaker release (rest, core):

```javascript
Projects.create({
      path,
      name,
      namespaceId: parentGroupId,
      defaultBranch: "main",
      initializeWithReadme: true,
    }
```
Whichever comes second (path or name) gets linted as an error because the type def says:
```javascript
 OneOf<{
        name: string;
        path: string;
    }>
 ```
The [API Docs](https://docs.gitlab.com/ee/api/projects.html#create-project) show that both are allowed to be specified, but ATLEAST one of the two is required.

**Checklist**

- [X] I have checked that this is not a duplicate issue.
- [X] I have read the documentation.

**Comment by jdalrymple** [comment]
Waiting for deployment - An unrelated job is failing -_-

**Comment by jdalrymple** [comment]
:rocket: Issue was released in [`39.34.1`](https://github.com/jdalrymple/gitbeaker/releases/tag/39.34.1) :rocket:
